index.html

```diff
@@ -15,15 +15,18 @@
 <body>
     <nav class="navbar">
         <h2 class="logo"><a href="/index.html">Home</a></h2>
+        
 
         <div class="input-container">
             <i class="fas fa-search icon"></i>
             <input class="input-field" type="text" placeholder="Search" />
         </div>
 
         <div class="top-links">
+            <h2 class="btn"><a href="/signUpPage/signup.html">Signup</a></h2>
+            <h2 class="btn"><a href="/logoutPage/logout.html">logout</a></h2>
             <h2 class="btn"><a href="/product.html">Product</a></h2>
-            <h2 class="btn"><a href="#">Login</a></h2>
+            <h2 class="btn"><a href="/signInPage/signin.html">Login</a></h2>
 
             <div class="child-ecom">
                 <button onclick="location.href = '/cart/cart.html';" type="button" class="icon-button"><i class="fas fa-shopping-cart fa-3x"></i>
```

Comment:
You can also use an Anchor tag and make it look like a button by using your button classes because it's eventually acting just as a link to another page. 

You can add a <a> tag and add all the classes  that are applied to <button> and <i> tag